### PR TITLE
feat(CPTS): add new resource-CPTS_project

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:
1. add a new resource CPTS project

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/cpts' TESTARGS='-run=TestAccProject_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cpts -v -run=TestAccProject_basic -timeout 360m -parallel 4
=== RUN   TestAccProject_basic
--- PASS: TestAccProject_basic (17.81s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cpts      17.887s
```

### CODE REVIEW

File: huaweicloud/services/acceptance/cpts/resource_huaweicloud_cpts_project_test.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package dew
```
Comment (niuzhenguo) [inline huaweicloud/services/acceptance/cpts/resource_huaweicloud_cpts_project_test.go]: s/dew/cpts

Comment (hmbig2) [reply huaweicloud/services/acceptance/cpts/resource_huaweicloud_cpts_project_test.go]: done

File: huaweicloud/services/cpts/resource_huaweicloud_cpts_project.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+			Name:        d.Get("name").(string),
+			Description: utils.String(d.Get("description").(string)),
+		},
+	}
+	response, err := client.CreateProject(createOpts)
+	if err != nil {
+		return diag.Errorf("error creating CPTS project: %s", err)
+	}
+
+	d.SetId(fmt.Sprint(*response.ProjectId))
```
Comment (ShiChangkuo) [inline huaweicloud/services/cpts/resource_huaweicloud_cpts_project.go]: 1. should check whether the field is a nil pointer;
2. suggest to use strconv.Itoa

Comment (hmbig2) [reply huaweicloud/services/cpts/resource_huaweicloud_cpts_project.go]: done

**Pull request merged**